src/main.cpp

```diff
@@ -275,51 +275,55 @@ bool GetTransaction(const uint256 &hash,
     CTransactionRef &txOut,
     const Consensus::Params &consensusParams,
     uint256 &hashBlock,
-    bool fAllowSlow)
+    bool fAllowSlow,
+    const CBlockIndex *blockIndex)
```

Comment:
nit: I found useful to add a something like that

```c++
const CBlockIndex *blockIndex /* = nullptr */)
```

in function definition when it comes to optional parameter.